api/core/key_test.go

```diff
@@ -1,31 +1,33 @@
-package core
+package core_test
```

Comment:
suggestion: use `core` package name

Reply:
By using `core_test` we ensure that we're only testing public interfaces.

Reply:
☝️